### DISCUSSION TITLE
About Google Game Builder

### DISCUSSION THREAD

**Discussion by jtopaz1** [reZach/blog#5] [General]
Completed

**Comment by reZach** [comment]
Hi @jtopaz1, 

Thanks for reaching out. Yes it is true that I used to enjoy that game and work and make tools and scripts for it. Unfortunately yes it was closed down, for it must have been a lower than expected popularity for the game and the Google team open sourced it and went to other projects.

I'll be frank with you, I enjoyed the game but do no longer have desire to tinker around with the game like I used to. I do not have enough interest in this to step into another side project, life is just too busy now.

If it's a script you already have written I can probably dedicate a short review to point you in the right direction but I have no interest in restarting the open source GameBuilder or providing any support or tooling as I have done in the past. I want to be honest and clear with you here.

**Reply by jtopaz1** [reply]
.

**Reply by reZach** [reply]
I suggest you post an issue [here](https://github.com/jannes-io/steam-workshop-downloader) with this question. You really have questions about that repository working with delisted steam games. I found the app id you can use in it's config: 929860 (https://steamdb.info/app/929860/). For the workshopItem section you'll need something like this:

```
"workshopItems":{
    "a": 1570012381,
    "b": 1576070425,
    // etc.
}
```

(The tool could probably work with just these options though.. not sure as I haven't tested it).

The issue is that you'll need to scrape all workshop item ids, which [based on this post](https://www.reddit.com/r/CitiesSkylinesModding/comments/n63fvj/how_to_get_steam_ids_for_all_published_mods/) involves some code (or web scraping - which may or may not be acceptable use in the Steam terms of service), but if you aren't already familiar with that, the next best option is probably to copy/paste the id from the url from the [search page](https://steamcommunity.com/workshop/browse/?appid=929860&browsesort=trend&section=readytouseitems&days=90&actualsort=trend&p=1#).

**Reply by jtopaz1** [reply]
.

**Comment by reZach** [comment]
@jtopaz1 No sorry, I don't have any of those still saved on my computer

**Reply by jtopaz1** [reply]
.

**Reply by reZach** [reply]
Hi @jtopaz1.

I hope you respect my decision, but I am not comfortable sharing my steam account to someone I do not know.
For this reason, I suggest you may try reaching out to one of the other two contributors on the [gamebuilder-plus repo](https://github.com/orangy/gamebuilder-plus). Additionally, the team who built the game may be able to help, you can reach out to them [at the archived repo](https://github.com/googlearchive/gamebuilder) or email them at gamebuilder@google.com.

**Reply by jtopaz1** [reply]
Hi,
I understand your difficulty. Thank you very much for the suggestion :)
Best

**Reply by jtopaz1** [reply]
.

**Reply by jtopaz1** [reply]
.

**Reply by reZach** [reply]
Hi @jtopaz1. Thank you for offering, but I am not interested in an outsourcing job. These days, I am busy doing other things and don't have that same interest in video games or Game Builder as I once had. Any programmer who is familiar with JSON would be able to help you out - so I wish you luck in finding someone!
